### PR TITLE
Add EIP-6036: Subscribeable NFT Extension

### PR DESCRIPTION
**Pull request by aboutmydreams**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

File: EIPS/eip-6036.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
+---
+eip: 6036
+title: Subscribeable NFT Extension
+description: 721sub is a subscribable nft protocol that allows users to enjoy their benefits for a certain period of time.
+author: Diven (@aboutmydreams), Kaso (@kasoqian), Jason (@350306878)
+discussions-to: https://github.com/ethereum/EIPs/pull/6036
```
Comment (Pandapip1) [inline EIPS/eip-6036.md:6]: This should be a thread on FEM

File: EIPS/eip-6036.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+  When an equity manager can be subscribed, it has an expiration time. We need an interface to determine whether the user's equity is within the time range that can be implemented.
+
+## Backwards Compatibility
+
+This is a new token type and is not meant to be backward compatible with any existing tokens other than existing viable souls like [EIP-721](./eip-721.md) tokens.
+
+
+## Reference Implementation
+
+```
```
Comment (Pandapip1) [inline EIPS/eip-6036.md:72]:
```suggestion
```solidity
```

File: EIPS/eip-6036.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
+---
+eip: 6036
+title: Subscribeable NFT Extension
+description: 721sub is a subscribable nft protocol that allows users to enjoy their benefits for a certain period of time.
```
Comment (Pandapip1) [inline EIPS/eip-6036.md:4]: This needs to be more tecnical and more descriptive. Did you mean:
```suggestion
description: An interface that allows for temporary ownership of EIP-721 NFTs
```

File: EIPS/eip-6036.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
+---
+eip: 6036
+title: Subscribeable NFT Extension
```
Comment (Pandapip1) [inline EIPS/eip-6036.md:3]:
```suggestion
title: Subscribe-able NFTs
```
Subscribeable is not a word. We discourage the use of the word "extension"

File: EIPS/eip-6036.md [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+interface IERC6036 {
+    event TokenIsExpire(uint indexed tokenId);
+
+    function tokenSubscribeExtend(uint _tokenID, uint _time) external payable;
+
+    function tokenSubscribeRevoke(uint _tokenID) external;
+
+    function isTokenExpire(uint _tokenID) external returns (bool);
+
+    function queryTokenExpire(uint _tokenID) external returns(uint);
```
Comment (Pandapip1) [inline EIPS/eip-6036.md:34]: How about these names (the names you have right now are grammatically incorrect):
```suggestion
    event TokenExpire(uint indexed tokenId);

    function extendTokenSubscription(uint _tokenID, uint _time) external payable;

    function revokeTokenSubscription(uint _tokenID) external;

    function hasExpired(uint _tokenID) external returns (bool);

    function tokenExpiration(uint _tokenID) external returns (uint);
```

File: EIPS/eip-6036.md [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+Subscribeable NFT Extension is a standard that allows more platforms to directly use the interface to determine whether a user is still within the enjoyment time of a certain benefit
+
+## Motivation
+
+Web2 subscriptions are leased, not proprietary. You don't own a membership to your streaming audio subscription site Spotify, so you can't sell that membership. Since Apple takes a 30% commission from content providers, you're actually paying Apple's "platform tax." But this is just a preliminary idea, the following functions are what subscription NFTs can really do.
+
+
+## Specification
+
+```
```
Comment (Pandapip1) [inline EIPS/eip-6036.md:24]:
```suggestion
```solidity
```

File: EIPS/eip-6036.md [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+
+    function tokenSubscribeRevoke(uint _tokenID) external;
+
+    function isTokenExpire(uint _tokenID) external returns (bool);
+
+    function queryTokenExpire(uint _tokenID) external returns(uint);
+} 
+```
+
+`TokenIsExpire` means the expiration of a token, this is an event that should be thrown when the token expires.
```
Comment (Pandapip1) [inline EIPS/eip-6036.md:38]: How do you propose this is emitted?

File: EIPS/eip-6036.md [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+discussions-to: https://github.com/ethereum/EIPs/pull/6036
+status: Draft
+type: Informational
+created: 2022-08-17
+requires: 165, 721
+---
+
+## Abstract
+
+Subscribeable NFT Extension is a standard that allows more platforms to directly use the interface to determine whether a user is still within the enjoyment time of a certain benefit
```
Comment (Pandapip1) [inline EIPS/eip-6036.md:15]: This is unclear. I think you meant:
```suggestion
This EIP standardizes an interface extending [EIP-721](./eip-721.md) that allows for NFTs to be temporarily purchased.
```
This still needs to be a bit longer, though.

File: EIPS/eip-6036.md [hunk 9]
```diff
@@ -0,0 +1,116 @@
...
+
+## Motivation
+
+Web2 subscriptions are leased, not proprietary. You don't own a membership to your streaming audio subscription site Spotify, so you can't sell that membership. Since Apple takes a 30% commission from content providers, you're actually paying Apple's "platform tax." But this is just a preliminary idea, the following functions are what subscription NFTs can really do.
```
Comment (Pandapip1) [inline EIPS/eip-6036.md:19]: I can't really understand what you're saying here. Would you mind rephrasing this for me?

**Review by Pandapip1 — CHANGES_REQUESTED** [review]
Initial review. There are still other things that need fixing.

File: EIPS/eip-6036.md (hunk 5 above)
Comment (aboutmydreams) [reply EIPS/eip-6036.md:34]: thanks, I think these modifications are more explicit

File: EIPS/eip-6036.md (hunk 9 above)
Comment (kasodesyn) [reply EIPS/eip-6036.md]: thanks for your suggestion, I rewrite this.

File: EIPS/eip-6036.md (hunk 1 above)
Comment (aboutmydreams) [reply EIPS/eip-6036.md]: thanks, updated

**Comment by aboutmydreams** [comment]
@Pandapip1 We have fixed the issues mentioned above. Thanks for checking again

**Pull request closed without merging**